### PR TITLE
Aladin target overlay using proper motion

### PR DESCRIPTION
**Pull request by cquiroz**
This PR will show the offsets (For now simulated) on the aladin component.
It will also use them for AGS and it includes a user preference to select if the offsets are displaed or not

image

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala [hunk 1]
```diff
@@ -60,7 +60,20 @@ case class AladinContainer(
   updateViewOffset:       Offset => Callback,
   selectedGuideStar:      Option[AgsAnalysis],
   guideStarCandidates:    List[AgsAnalysis]
-) extends ReactFnProps(AladinContainer.component)
+) extends ReactFnProps(AladinContainer.component) {
+  // ABBA offset
+  def offsets: List[Offset] = List(
```
[the remark this answers is not in the document]
Comment (cquiroz) [reply explore/src/main/scala/explore/targeteditor/AladinContainer.scala]: Right, this was to make sure than offsets on p are properly handled but I could switch to the default

File: common-graphql/src/main/resources/graphql/schemas/UserPreferencesDB.graphql [hunk 2]
```diff
@@ -344,6 +344,7 @@ input ExplorePlotTimeUpdates {
 columns and relationships of "exploreTargetPreferences"
 """
 type ExploreTargetPreferences {
+  acquisitionOffsets: Boolean
```
Comment (rpiaggio) [inline common-graphql/src/main/resources/graphql/schemas/UserPreferencesDB.graphql:347]: Is this temporary while they are not in the API?

Comment (cquiroz) [reply common-graphql/src/main/resources/graphql/schemas/UserPreferencesDB.graphql:347]: This a preference as to whether to display them or not, not related to the model

File: common/src/main/scala/explore/common/UserPreferencesQueries.scala [hunk 3]
```diff
@@ -232,32 +243,34 @@ object UserPreferencesQueries:
             )
             .getOrElse(Offset.Zero)
 
-          val agsCandidates = targetPrefsResult
-            .flatMap(_.agsCandidates)
-            .map(Visible.boolIso.get)
-            .getOrElse(Visible.Inline)
-          val agsOverlay    = targetPrefsResult
-            .flatMap(_.agsOverlay)
+          def visibleProp(op: ExploreTargetPreferencesByPk => Option[Boolean]) = targetPrefsResult
```
Comment (rpiaggio) [inline common/src/main/scala/explore/common/UserPreferencesQueries.scala:246]: Nice refactoring

**Review by rpiaggio — APPROVED** [review]
LGTM

**Comment by cquiroz** [comment]
Fixes [sc-1723]

**Pull request merged**